### PR TITLE
Doc update for V1-v2 apps

### PR DESCRIPTION
**Pull request by shivani-sumo**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [ ] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/saas-cloud/slack.md [hunk 1]
```diff
@@ -284,24 +284,17 @@ _sourceCategory=Labs/slack
...
-## Collect logs for the Slack App
+## Collect logs for the Slack app
 
 This legacy solution to pull logs from Slack to Sumo Logic has been replaced with a dedicated [Cloud-to-Cloud Integration Framework](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework). We recommend customers use the below [Slack Cloud-to-Cloud source](#create-slack-app-with-permissions) instead of the legacy Python collection method.
 
-## Installing the Slack App
+## Installing the Slack app
 
-This section demonstrates how to install the Slack App.
+{@import ../../reuse/filter-dashboards.md}
```
Comment (jpipkin1) [inline docs/integrations/saas-cloud/slack.md:293]:
```suggestion
{@import ../../reuse/app-install-v2.md}
```

**Review by jpipkin1 — APPROVED** [review]
@shivani-sumo, the build-and-deploy check on this PR failed, so I applied updates so the build would complete (see my commits).  Before you submit PRs in the future, [run the branches locally using yarn](https://github.com/SumoLogic/sumologic-documentation#building-locally) to ensure they build correctly.

File: docs/integrations/saas-cloud/slack.md [hunk 2]
```diff
@@ -284,24 +284,17 @@ _sourceCategory=Labs/slack
...
-## Collect logs for the Slack App
+## Collect logs for the Slack app
 
 This legacy solution to pull logs from Slack to Sumo Logic has been replaced with a dedicated [Cloud-to-Cloud Integration Framework](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework). We recommend customers use the below [Slack Cloud-to-Cloud source](#create-slack-app-with-permissions) instead of the legacy Python collection method.
 
-## Installing the Slack App
+## Installing the Slack app
 
-This section demonstrates how to install the Slack App.
+{@import ../../reuse/app-install-v2.md}
```
Comment (jpipkin1) [inline docs/integrations/saas-cloud/slack.md:293]:
```suggestion
{@import ../../reuse/apps/app-install-v2.md}
```

File: docs/integrations/saml/onelogin.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ Once you begin uploading data, your daily data usage will increase. It's a good
    * From OneLogin, configure a broadcaster that points to this endpoint using the instructions in the [OneLogin documentation](https://onelogin.service-now.com/support?id=kb_article&sys_id=43f95543db109700d5505eea4b961959). You must use SIEM (NDJSON) format. Use the Sumo Logic HTTP Source URL as the Listener URL, and custom header is not needed.
 
 
-## Log Types and Sample Log Message
+## Log Types and sample log message
```
Comment (kimsauce) [inline docs/integrations/saml/onelogin.md:30]: For consistency with the rest of the apps docs, Log types and Sample log messages should be separate sections.

**Review by kimsauce — CHANGES_REQUESTED** [review]
One change

**Comment by JV0812** [comment]
The intent of this PR is covered in the PRs #3432, #3433, and #3434.

**Pull request closed without merging**